okapi-common/src/main/java/org/folio/okapi/common/Config.java

```diff
@@ -68,4 +68,19 @@ public static Boolean getSysConfBoolean(String key, Boolean def, JsonObject conf
     return Boolean.parseBoolean(v);
   }
 
+  /**
+   * Returns integer config info from properties and JSON config.
+   * Check property first in system; if not found OR empty, inspect JSON configuration.
+   * @param key property key (JSON key).
+   * @param def default value (may be null).
+   * @param conf JSON object configuration.
+   * @return value (possibly null).
+   */
+  public static Integer getSysConfInteger(String key, Integer def, JsonObject conf) {
+    final String v = System.getProperty(key);
+    if (v == null || v.isEmpty()) {
+      return conf.getInteger(key, def);
+    }
+    return Integer.parseInt(v);
```

Comment:
Any concern if `v` cannot be parsed to integer?

Reply:
I'll check and, perhaps, make a test.

Reply:
I tried to the property to "foo" and Okapi will fail the deployment and return a nice 400. And the log will include the backtrace ok. So I think we're good to go!